Mathlib/Algebra/BigOperators/Basic.lean

```diff
@@ -1858,6 +1858,11 @@ theorem prod_update_of_mem [DecidableEq α] {s : Finset α} {i : α} (h : i ∈
 #align finset.prod_update_of_mem Finset.prod_update_of_mem
 #align finset.sum_update_of_mem Finset.sum_update_of_mem
 
+@[to_additive]
+theorem prod_update_one_of_mem [DecidableEq α] {s : Finset α} {i : α} (hi : i ∈ s) (f : α → β) :
+    ∏ j in s, Function.update f i 1 j = ∏ j in s \ singleton i, f j := by
```

Comment:
```suggestion
    ∏ j in s, Function.update f i 1 j = ∏ j in s.erase i, f j := by
```
This is the canonical spelling. Though I see `prod_update_of_mem` also doesn't use this, so maybe ignore this comment.

Reply:
Thanks!